base_phone_validation/README.rst

```diff
@@ -0,0 +1,8 @@
+Base Phone Validation
```

Comment:
As the README misses some parts, you can take the occasion for using the new fragments system. Check OCA/maintainer-tools for the template